lib/util.js

```diff
@@ -22,6 +22,17 @@ const fs = require('fs');
  */
 function createReadStream(path, options) {
     return new Promise((resolve, reject) => {
+        if(typeof options === 'string' || options instanceof String){
+            const encoding = options;
+            options = {};
+            options.encoding = encoding;
```

Comment:
I think you can simplify this to:

```suggestion
            options = { encoding: options };
```